style_editor.html

```diff
@@ -97,6 +100,97 @@
 input[type="button"]:active {
   background-color: dodgerblue;
 }
+
+.icon-button {
+  position: relative;
+}
+
+.icon-button .fa-flip {
+  animation: none;
+}
+.icon-button:hover .fa-flip {
+  animation: flip-once 0.75s ease forwards;
+}
+@keyframes flip-once {
```

Comment:
Whut?

Reply:
yeah! Check them out
https://nosloppy.github.io/ProffieOS-StyleEditor-1/style_editor_BC_V5.html

Reply:
No.
There is a reason the <blink> tag doesn't exist anymore.

Reply:
the < blink > tag

Reply:
Aww. THis is the dialed back version! Initially, they would just animate on loop while hovered, but I did realize that's likely too distracting, so I went with the one shot instead. Still too much? Brings some life to them I think.
Your call of course. I can always have a personal version :)

Reply:
how about a settings button? Give the user the choice?

Reply:
No.

Reply:
ok. they're plain jane now.

Reply:
> No.

Would you let the just the Rotate button animate slowly while active?

Reply:
no.